be/src/runtime/routine_load/data_consumer_pool.cpp

```diff
@@ -68,11 +68,17 @@ Status DataConsumerPool::get_consumer_grp(StreamLoadContext* ctx,
     }
     DCHECK(ctx->kafka_info);
 
+    if (ctx->kafka_info->begin_offset.size() == 0) {
+        return Status::InternalError(
```

Comment:
This is not an error, if a pipe has not data for a long time, it will write a lot of error logs.
Check it before call get_consumer_grp maybe better. Otherwise check return value and don't write warning log each time.